<ui version="4.0" >
 <class>FormScreenshot</class>
 <widget class="QDialog" name="FormScreenshot" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Screenshot Dialog Box</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="glnemo.qrc" >:/images/camera.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="res_box" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Resolution</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="method_screen" >
        <property name="toolTip" >
         <string>Select resolution from the OpenGL window</string>
        </property>
        <property name="text" >
         <string>screen</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="method_standard" >
        <property name="toolTip" >
         <string>Select resolution from standard value</string>
        </property>
        <property name="text" >
         <string>standard</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="method_custom" >
        <property name="toolTip" >
         <string>Select resolution from custom values</string>
        </property>
        <property name="text" >
         <string>custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="std_box" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string/>
     </property>
     <property name="title" >
      <string>Standard</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="stand_320" >
        <property name="text" >
         <string>320x240</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="stand_640" >
        <property name="text" >
         <string>640x480</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="stand_800" >
        <property name="text" >
         <string>800x600</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="stand_1024" >
        <property name="text" >
         <string>1024x768</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="stand_1152" >
        <property name="text" >
         <string>1152x864</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QRadioButton" name="stand_1280" >
        <property name="text" >
         <string>1280x1024</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="custom_box" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="title" >
      <string>Custom</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="nbody_edit" >
        <property name="text" >
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="custom_w" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="custom_h" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Height</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="shot_button" >
       <property name="text" >
        <string>Shot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="glnemo.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>FormScreenshot</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>263</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel" >
     <x>273</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method_screen</sender>
   <signal>clicked(bool)</signal>
   <receiver>std_box</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>21</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method_screen</sender>
   <signal>clicked(bool)</signal>
   <receiver>custom_box</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>61</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method_standard</sender>
   <signal>clicked(bool)</signal>
   <receiver>std_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>139</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>114</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method_standard</sender>
   <signal>clicked(bool)</signal>
   <receiver>custom_box</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>143</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>143</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method_custom</sender>
   <signal>clicked(bool)</signal>
   <receiver>std_box</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>193</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>186</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method_custom</sender>
   <signal>clicked(bool)</signal>
   <receiver>custom_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>225</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>219</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
